code/go/0chain.net/smartcontract/dbs/event/process.go

```diff
@@ -27,6 +27,7 @@ const (
 	TagAddOrOverwriteBlobber
 	TagUpdateBlobber
 	TagDeleteBlobber
+	TagAddTransaction
```

Comment:
Where is this being used? According to current structure, there is no TagAddTransaction being emitted

Reply:
done

Reply:
done... discussed and a different approach was decided and implemented